etc/frontend_development.ini.in

```diff
@@ -28,6 +28,14 @@ adhocracy.frontend.template_path = /static/templates
 adhocracy.frontend.rest_url = http://localhost:6541
 # backend behind varnish (default port)
 #adhocracy.frontend.rest_url = http://localhost:8088
+# Only accept registration requests with valid captcha solutions
+adhocracy.captcha.required = ${thentos_captcha:required}
+# Where the frontend sends captcha traffic
+adhocracy.captcha.backend_url = ${thentos_captcha:backend_url}
+# Where the backend sends captcha traffic (in development mode, this
+# is the same as captcha backend; if nginx is dispatching inbound
+# requests, this is just the frontend url)
+adhocracy.captcha.frontend_url = ${thentos_captcha:frontend_url}
 # The email address show to users seeking support
```

Comment:
i will do that.

i'm worried that this will create a lot of boilerplate that is tedious to maintain, but i'm assuming this is a bigger subject than this pull request.